packages/flutter_tools/lib/src/context_runner.dart

```diff
@@ -204,6 +204,7 @@ Future<T> runInContext<T>(
           platform: globals.platform,
           featureFlags: featureFlags,
         ),
+        fuchsiaSdk: globals.fuchsiaSdk,
```

Comment:
A lot of these `globals` instances are nullable, but are only ever null from tests.  This library is going to be annoying to migrate later...